### PR TITLE
Add an issue template to provide some framing for new issues.

### PR DESCRIPTION
**Pull request by romeojulietthotel**
Issue templates should do two things. _One_ is to act as a guide for the user that is reporting the issue and _two_ to provide some useful data for a busy maintainer. It is meant to start the conversation off with just the right amount of momentum and is not a burden to the reporter.

### CODE REVIEW

**Comment by jpturcotte** [comment]
Sorry for the duplicate PR! Might I suggest adding my proposed header and Prerequisites section, though?

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+### --System Information--
++ OpenToonz Version:
+<!-- You can find the VERSION in Menu->Help->About OpenToonz -->
+
++ Operating System:
+<!-- (e.g., Windows10, macOS 10.12, Ubuntu 16.04, Debian Stretch) -->
+
++ RAM Size:
+<!-- (4GB, 8GB etc.) -->
```
Comment (shun-iwasawa) [inline .github/ISSUE_TEMPLATE.md]: In my experience it was rare that the issue was proved to be due to the memory shortage so far.
How about making this item optional, or combining with the following `Graphics Tablet` item and rename like `Other Device Info` which is for users to write down any optional device information?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
++ Graphics Tablet:
+<!-- (Optional) (Write your MODEL, e.g., WACOM Intuos 2) -->
+
+### --Actual Results--
+<!-- Explain what you actually got. -->
+
+### --Expected Results--
+<!-- Explain how the program should behave in the context you're using it. -->
```
Comment (shun-iwasawa) [inline .github/ISSUE_TEMPLATE.md]: How about moving `Actual Results` and `Expected Results` to just under `Step to reproduce` , since they are related?

Comment (romeojulietthotel) [reply .github/ISSUE_TEMPLATE.md]: I had those last as sometimes they become quite long and the other info can provide a quick summary without having to look over long cut-n-pasted fields. It's a minor consideration and is easily changed back if cut-n-pastes happen often.

**Comment by shun-iwasawa** [comment]
LGTM Thank you @romeojulietthotel !

**Pull request merged**